dev-lua/lgi/lgi-0.9.2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VIRTUALX_REQUIRED="manual"
+
+inherit eutils toolchain-funcs flag-o-matic virtualx
+
+DESCRIPTION="Lua bindings using gobject-introspection"
+HOMEPAGE="https://github.com/pavouk/lgi"
+SRC_URI="https://github.com/pavouk/lgi/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+IUSE="examples test"
+
+RDEPEND=">=dev-lang/lua-5.1:=
+		dev-libs/gobject-introspection
+		dev-libs/glib
+		virtual/libffi:0="
+DEPEND="${RDEPEND}
+		virtual/pkgconfig
```

Comment:
````virtual/pkgconfig``` is ```BDEPEND```

Reply:
thank you, still getting used to BDEPEND